### ISSUE TITLE
Try discrete jumps in learning rate

### ISSUE THREAD

**Issue by awohns** [awohns/tspyro#31]
One suggestion from @martinjankowiak is that the learning rate decay we currently use could be overkill. We should try out dropping the learning rate in discontinuous jumps (i.e. after 1000, 2000 steps etc.).

**Comment by martinjankowiak** [comment]
this is done like this:

```
from pyro.optim import MultiStepLR
scheduler = MultiStepLR({'optimizer': torch.optim.Adam,
                         'optim_args': {'lr': 0.001},
                         'gamma': 0.2, 'milestones': [100, 1000]})
SVI(..., scheduler, ...)
```
